README.md

```diff
@@ -1,13 +1,15 @@
 # react-media-query-hoc
-A dead simple React Higher Order Component (HOC) that uses context for matching media queries
+A dead simple React Higher Order Component (HOC) that uses context for matching media queries.
 
 
 ## Why use this?
 - A simple API which doesnt require you to put `MediaQuery` components all over your code base
 - More performant (you only need 1 parent `MediaQueryProvider` that listens to media events you wish to configure)
 - Easier to test than other react media query libraries
-- Uses [matchmedia](https://github.com/iceddev/matchmedia) for media queries for client and server
-- Abstracted away React context which is experimental (and subject to change for <= React 15)
+- Uses [css-mediaquery](https://github.com/ericf/css-mediaquery) for server side rendering
+
+## Why not use this?
+We always recommend using vanilla CSS media queries to build responsive websites, this is simpler and provides a smoother UX, also it mitigates having to guess the screen width during [server side rendering](#server-side-rendering). At Domain we needed to use this component for legacy ad tech and stat reasons and advise against it's use for general responsive website design.
```

Comment:
"We recommend using vanilla CSS media queries to build responsive websites over JS solutions, this is simpler,  provides a smoother UX and also it mitigates having to guess the screen width during [server side rendering](#server-side-rendering). 
Use this library if you need to dramatically alter the page layout between media types (a good example would be if you had to render completely different components)"